readme/Configuration.scalatex

```diff
@@ -423,6 +423,27 @@
       @b{NEW}. Semantically preserving rewrites. To enable a
       rewrite rule, add it to the config like this @b{rewrite.rules = [SortImports]}.
 
+    @sect(Rewrite.rewrite2name(AvoidInfix))
+      @demoStyle(rewriteInfix)
+        future recover {
+          case e: ArithmeticException ⇒ 0
+        } map (_.toString)
+
+    @sect(Rewrite.rewrite2name(ExpandImportSelectors))
+      @demoStyle(rewriteImportSelectors)
+        import a.{
+            b,
+            c
+          }, h.{
+            k, l
+          }
+          import d.e.{f, g}
+          import a.{
+              foo => bar,
+              zzzz => _
```

Comment:
Thanks! I bumped into this recently while implementing "organize imports" in scalafix. Didn't know renames and unimports needs to be bundled together with the wildcard.